loginsightwebhookdemo/__init__.py

```diff
@@ -251,8 +258,17 @@ def _introduction():
 @app.route("/endpoint/test", methods=['POST'])
 @app.route("/endpoint/test/<ALERTID>", methods=['POST'])
 def test(ALERTID=None):
-    """Log the request and respond with success. Don't send the payload anywhere."""
+    """Log the auth header, request, and parsed moreinfo field. Respond with success. Don't send the payload anywhere."""
+    try:
+        logging.info(request.headers['Authorization'])
+    except KeyError:
+        pass
     logging.info(request.get_data())
+    try:
+        a = parse(request)
```

Comment:
This wasn't in your `try` block before, and probably shouldn't be -- if the body can't be parsed then raising an exception is appropriate; falling through to `return "OK"` isn't.

Reply:
The only reason to parse is for print on line 269. If either fail, it is OK (parse is tested separately already). What is the concern with including it in the try?

Reply:
Including it hides other (possibly future) problems.

Reply:
OK, will update momentarily.